lib/sensors/accelerometer.cpp

```diff
@@ -0,0 +1,117 @@
+#include "accelerometer.hpp"
+
+namespace hyped::sensors {
+
+Accelerometer::Accelerometer(std::uint8_t mux_address_ess, io::I2c &i2c, core::ILogger &log)
+    : address_(mux_address_ess),
+      log_(log),
+      i2c_(i2c)
+{
+}
+
+Accelerometer::~Accelerometer()
+{
+}
+
+std::optional<std::int16_t> Accelerometer::getRawAccelerationX()
+{
+  std::optional<std::uint8_t> resultLow  = i2c_.readByte(address_, 0x28);
+  std::optional<std::uint8_t> resultHigh = i2c_.readByte(address_, 0x29);
```

Comment:
also, for the addresses of all the registers we read from - could you `static constexpr` them with just the actual register name, e.g `X_OUT_L = 0x28`. Just so we don't have any magic numbers in the code and the register names can easily be cross-referenced with the datasheet.

Reply:
`kXOutL` for style reasons